docs/explanations/flyscanning.rst

```diff
@@ -1,42 +1,6 @@
 Flyscanning
 ===========
 
-Flyscanning (also known as hardware triggered scanning, asynchronous acquisition, and hardware synchronized scanning) is the practice of accelerating the rate of data collection by handing control over to an external hardware system that can control and synchronize the triggering of detectors with other signals and/or commands. Flyscans take many forms.
```

Comment:
Will have to wait until after https://github.com/bluesky/bluesky-cookbook/pull/12 is merged, but I think we should then link to that page here